dim-testsuite/t/layer3domain-pool-list.t

```diff
@@ -0,0 +1,73 @@
+$ ndcli create layer3domain l3d-test-1 type vrf rd 0:0
+
+$ ndcli create container 10.0.0.0/8 layer3domain l3d-test-1
+INFO - Creating container 10.0.0.0/8 in layer3domain l3d-test-1
+$ ndcli create container 10.0.0.0/8 layer3domain default
+INFO - Creating container 10.0.0.0/8 in layer3domain default
+
+$ ndcli create container 172.16.0.0/12 layer3domain l3d-test-1
+INFO - Creating container 172.16.0.0/12 in layer3domain l3d-test-1
+$ ndcli create container 172.16.0.0/12 layer3domain default
+INFO - Creating container 172.16.0.0/12 in layer3domain default
+
+$ ndcli create pool test-pool-1 layer3domain l3d-test-1
+$ ndcli create pool test-pool-default layer3domain default
+
+$ ndcli modify pool test-pool-1 add subnet 10.0.0.0/24 layer3domain:l3d-test-1
+INFO - Created subnet 10.0.0.0/24 in layer3domain l3d-test-1
+WARNING - Creating zone 0.0.10.in-addr.arpa without profile
+WARNING - Primary NS for this Domain is now localhost.
+$ ndcli modify pool test-pool-default add subnet 10.1.0.0/24 layer3domain:default
+INFO - Created subnet 10.1.0.0/24 in layer3domain default
+WARNING - Creating zone 0.1.10.in-addr.arpa without profile
+WARNING - Primary NS for this Domain is now localhost.
+
+$ ndcli modify pool test-pool-1 add subnet 172.16.0.0/24 layer3domain:l3d-test-1
+INFO - Created subnet 172.16.0.0/24 in layer3domain l3d-test-1
+WARNING - Creating zone 0.16.172.in-addr.arpa without profile
+WARNING - Primary NS for this Domain is now localhost.
+$ ndcli modify pool test-pool-default add subnet 172.17.0.0/24 layer3domain:default
+INFO - Created subnet 172.17.0.0/24 in layer3domain default
+WARNING - Creating zone 0.17.172.in-addr.arpa without profile
+WARNING - Primary NS for this Domain is now localhost.
+
+
+$ ndcli list pools
+name              vlan subnets
```

Comment:
Should the layer3domain be added to the default output columns?

Reply:
Yes, that's better.